### DISCUSSION TITLE
Voice detection

### DISCUSSION THREAD

**Discussion by Etsi-o** [Audiveris/audiveris#755] [General]
Hi, Im also quite new to the audiveris and also very impressed!
Is audiveris able to identify / distinguish between different text meanings?
Currently there is a distingtion between Lyrics and System text, however there is also text such as bar-numbering, instrument name, etc. which would be great if the engine were able to identify.
Are there any way to specify this or plans on making it able to?

I would love to help this project in the ways im able (without any particular coding experience). How could someone like me help?
Also, iv'e read that you are looking at improving the training dataset by using real life examples with the "Annotate Book Symbols" in the handbook, however this was a bit dificult to understand.

**Comment by hbitteur** [comment]
I would suggest that you first read the [Audiveris handbook](https://audiveris.github.io/audiveris/_pages/handbook/).
There you should discover that a sentence can be assigned a "role" either by the engine or by the end user.
